src/storage/src/hummock/event_handler/hummock_event_handler.rs

```diff
@@ -465,67 +465,14 @@ impl HummockEventHandler {
             .start_sync_epoch(new_sync_epoch, sync_result_sender, table_ids);
     }
 
-    async fn handle_clear(&mut self, notifier: oneshot::Sender<()>, version_id: HummockVersionId) {
+    fn handle_clear(&mut self, notifier: oneshot::Sender<()>) {
         info!(
-            ?version_id,
             current_version_id = ?self.uploader.hummock_version().id(),
             "handle clear event"
         );
 
         self.uploader.clear();
 
-        let current_version = self.uploader.hummock_version();
-
-        if current_version.id < version_id {
-            let mut latest_version = if let Some(CacheRefillerEvent {
-                pinned_version,
-                new_pinned_version,
-            }) = self.refiller.clear()
```

Comment:
Just a note: after this PR, the refiller buffer won't get cleared on `handle_clear`. This won't affect correctness but may slightly delay future version update because a new version update needs to wait for previous refiller events to complete before it can be effective.

Reply:
How come we no longer need to clear refiller?

Reply:
The refiller was previously clear for the logic of waiting for receiving the latest hummock version. The event handler should wait for receiving the latest hummock in meta node, so that the newly spawned actors can see the latest hummock version after recovery. However, after this PR, the spawned actors will explicitly wait for the `committed_epoch` of the state table to bump up to the `prev_epoch` in `init_epoch`, so that we don't need to wait for the latest hummock version globally during recovery.